Texture.podspec

```diff
@@ -14,10 +14,7 @@ Pod::Spec.new do |spec|
   spec.weak_frameworks = 'Photos','MapKit','AssetsLibrary'
   spec.requires_arc = true
 
-  spec.ios.deployment_target = '8.0'
-
-  # Uncomment when fixed: issues with tvOS build for release 2.0
-  # spec.tvos.deployment_target = '9.0'
```

Comment:
Let's not remove this yet - there are a few PRs pending with tvOS support, and there's still developer interest on that platform (besides other folks who have asked for it, I also have plans for a work project).